# srv file for service to check order completion

string order_id
bool completion_status 

---

bool success